packages/fether-electron/src/main/index.js

```diff
@@ -57,12 +58,33 @@ function createWindow () {
         parityChannel: parity.channel
       })
     )
-    .then(() =>
+    .then(async () => {
       // Run parity when installed
-      runParity(['--light', '--chain', cli.chain || 'kovan'], err =>
-        handleError(err, 'An error occured with Parity.')
-      )
-    )
+
+      // Don't run parity if the user ran fether with --no-run-parity
+      if (!cli.runParity) {
+        return;
+      }
+
+      if (await isParityRunning({
+        wsInterface: cli.wsInterface,
+        wsPort: cli.wsPort
+      })) {
+        return;
+      }
+
+      return runParity({
+        flags: [
+          ...getRemainingArgs(cli),
```

Comment:
I found one bug with `getRemainingArgs `. If I run fether with `fether --chain foundation`, then remainingArgs is `['foundation']`.